src/jvm/main/org/jetbrains/kotlinx/lincheck/strategy/native_calls/DeterministicRandomMethodDescriptor.kt

```diff
@@ -66,24 +71,24 @@ private fun MethodSignature.isSecureRandomMethodToSkip() = when (name) {
     else -> false
 }
 
-private fun ArgumentType.Object.isSecureRandom() = className == "java.security.SecureRandom"
+private fun ObjectType.isSecureRandom() = className == "java.security.SecureRandom"
 
-private val byteArrayMethodType = MethodType(
-    argumentTypes = listOf(ArgumentType.Array(ArgumentType.Primitive.Byte)),
-    returnType = Type.Void,
+private val byteArrayMethodType = Types.MethodType(
+    listOf(ArrayType(Types.BYTE_TYPE)),
+    Types.VOID_TYPE,
 )
 
-private val secureByteArrayMethodType = MethodType(
-    argumentTypes = listOf(
-        ArgumentType.Array(ArgumentType.Primitive.Byte),
-        ArgumentType.Object("java.security.SecureRandomParameters")
+private val secureByteArrayMethodType = Types.MethodType(
+    listOf(
+        ArrayType(Types.BYTE_TYPE),
+        ObjectType("java.security.SecureRandomParameters")
     ),
-    returnType = Type.Void,
+    Types.VOID_TYPE,
```

Comment:
Why removing argument names (`argumentTypes` and `returnType`)?
IMO argument names improve readability here.

Reply:
named arguments not allowed for java functions

Reply:
Oh, I see, sorry.